properties/P000203.md

```diff
@@ -0,0 +1,9 @@
+---
+uid: P000203
+name: Almost discrete
+---
```

Comment:
Given all the discussion, we should add a reference to justify why almost discrete doesn't imply discrete (i.e. that's how it's used in the literature).